app/controllers/katello/api/v2/hosts_bulk_actions_controller.rb

```diff
@@ -295,6 +301,10 @@ def find_deletable_hosts
       find_bulk_hosts(:destroy_hosts, params)
     end
 
+    def validate_organization
+      fail HttpErrors::BadRequest, _("Organization ID is required") if @organization.blank?
```

Comment:
`find_organization` will raise an error if it can't find the organization since you changed it to use `find` vs `find_by_id` which just returns nil if the org doesn't exist. This error wouldn't be raised as it is now. My recommendation is to change it back so that we aren't affecting the other APIs using the concern with this behavior change (even though we want the org_id to be required.), and perhaps file a separate issue that we can address next sprint or something.

Reply:
Thanks, didn't realize the error behavior was different.   Updating